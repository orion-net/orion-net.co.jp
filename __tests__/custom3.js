import React from "react"
import renderer from "react-test-renderer"

import Main from "../src/pages/custom3"

jest.mock("gatsby-plugin-image", () => {
  const StaticImageMock = () => <img>Mocked StaticImage component</img>

  return {
    StaticImage: StaticImageMock,
  }
})

describe("custom3", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Main />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
