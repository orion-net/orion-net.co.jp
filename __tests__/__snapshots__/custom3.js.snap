// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom3 renders correctly 1`] = `
Array [
  <div
    className="hamburger-menu"
  >
    <div
      className="bar-wrapper closed"
      onClick={[Function]}
    >
      <div
        className="bar bar0"
      />
      <div
        className="bar bar1"
      />
      <div
        className="bar bar2"
      />
    </div>
    <ul
      className="navbar closed"
      onClick={[Function]}
    >
      <li
        className=""
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          トップページ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/concept"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          サロン紹介
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/menu"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          メニュー
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          商品紹介
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom2"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          おすすめ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/custom1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          ご予約
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/access"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          アクセス
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/concept1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・七五三
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/contact"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          お問い合わせ
        </a>
      </li>
      <li
        className=""
      >
        <a
          href="/privacy"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        className="current_page_item"
      >
        <a
          href="/custom3"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          成人式・スナップ集
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="layout"
  >
    <div
      onClick={[Function]}
    >
      <div
        className="logo"
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          <img
            alt="オリオン美容室のトップ"
            height="353px"
            src="/img/top3.png"
            width="707px"
          />
        </a>
      </div>
      <div
        className="article-wrapper"
      >
        <div
          className="content-area"
          id="primary"
        >
          <div
            className="site-content sp-part-top sp-content page-custom3"
            id="content"
            role="main"
          >
            <header
              className="entry-header sp-part-top sp-page-title"
              id="sp-page-title-12"
            >
              <h1
                className="entry-title"
              >
                成人式・スナップ集
              </h1>
            </header>
            <div
              className="sp-part-top sp-block-container"
              id="sp-block-container-2"
            >
              <p
                className="large paragraph"
              >
                <b
                  className="character"
                  style={
                    Object {
                      "color": "#9e140c",
                    }
                  }
                >
                  ※転載厳禁
                </b>
              </p>
            </div>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <article>
              <div
                className="sp-part-top sp-block-container"
                id="page-content"
              >
                <a
                  style={
                    Object {
                      "display": "block",
                    }
                  }
                >
                  <img>
                    Mocked StaticImage component
                  </img>
                </a>
              </div>
            </article>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
            <a
              style={
                Object {
                  "display": "block",
                }
              }
            >
              <img>
                Mocked StaticImage component
              </img>
            </a>
          </div>
        </div>
      </div>
      <div
        className="sidebar"
      >
        <div>
          <a
            className="blog-button"
            href="https://ameblo.jp/teamorion/?frm_id=v.mypage-ameblo--myblog--blog"
          >
            ブログ
          </a>
        </div>
        <div>
          <a
            className="facebook-button"
            href="https://www.facebook.com/orionannex.orion"
          >
            Facebook
          </a>
        </div>
        <div
          className="sp-part-top sp-sns-button"
          id="sp-sns-button-1"
        >
          <ul>
            <li>
              <iframe
                allowtransparency="true"
                frameBorder="0"
                scrolling="no"
                src="https://www.facebook.com/plugins/like.php?href=http%3A%2F%2Forion-net.co.jp%2Fprivacy.html&layout=button_count&width=120&height=21&send=true&show_faces=true&share=false&action=like&colorscheme=light&font"
                style={
                  Object {
                    "border": "none",
                    "height": "21px",
                    "overflow": "hidden",
                    "width": "120px",
                  }
                }
              />
            </li>
            <li>
              <a
                className="twitter-share-button"
                data-count="horizontal"
                data-lang="ja"
                href="https://twitter.com/share"
              >
                Tweet
              </a>
              <script
                charSet="utf-8"
                src="https://platform.twitter.com/widgets.js"
                type="text/javascript"
              />
            </li>
            <li>
              <a
                href="https://line.me/R/msg/text/?%0D%0Ahttp%3A%2F%2Forion-net.co.jp%2Fprivacy.html"
              >
                <img
                  alt=""
                  height="20"
                  src="/img/linebutton_horizontal.png"
                  style={
                    Object {
                      "border": "none",
                    }
                  }
                  width="82"
                />
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  <footer
    className="footer"
  >
    <div
      className="footer-link"
    >
      <a
        className="privacy-policy-link"
        href="/privacy"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        プライバシーポリシー
      </a>
    </div>
    <p
      className="copyright"
    >
      Copyright © オリオン美容室, All rights reserved.
    </p>
  </footer>,
]
`;
